### PR TITLE
Add 7th Dragon III Code: VFD

### PR DESCRIPTION
**Pull request by jdenm8**

### CODE REVIEW

**Comment by jdenm8** [comment]
Just added the wiki article. Not too sure if the name for it is correct though.

File: games/7th-dragon-iii-code-vfd/game.dat [hunk 1]
```diff
@@ -0,0 +1,31 @@
+title = "7th Dragon III Code: VFD"
+description = "7th Dragon III: Code VFD is a JRPG developed and published by Sega. It is the third and final instalment in the 7th Dragon series, previously unreleased outside Japan."
+incomplete = false
```
Comment (Leo626) [inline games/7th-dragon-iii-code-vfd/game.dat]: This line isn't needed.

File: games/7th-dragon-iii-code-vfd/game.dat [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+github_issues = []
```
Comment (Leo626) [inline games/7th-dragon-iii-code-vfd/game.dat]: ditto

**Comment by jdenm8** [comment]
Done and done.

File: games/7th-dragon-iii-code-vfd/game.dat [hunk 3]
```diff
@@ -0,0 +1,29 @@
+title = "7th Dragon III Code: VFD"
+description = "7th Dragon III: Code VFD is a JRPG developed and published by Sega. It is the third and final instalment in the 7th Dragon series, previously unreleased outside Japan."
```
Comment (TheKoopaKingdom) [inline games/7th-dragon-iii-code-vfd/game.dat]: Inconsistent `;` in `7th Dragon III: Code VFD`.

Comment (TheKoopaKingdom) [inline games/7th-dragon-iii-code-vfd/game.dat]: `instalment` -> `installment`

File: games/7th-dragon-iii-code-vfd/game.dat [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+needs_system_files = false
+needs_shared_font = true
+
+[[releases]]
+title = "0004000000115400"
```
Comment (TheKoopaKingdom) [inline games/7th-dragon-iii-code-vfd/game.dat]: The USA release should go first (I realize this wasn't properly documented, it's been fixed.).

**Comment by jdenm8** [comment]
Fixed, Fixed and spelling changed to the US English one.

**Pull request merged**